pkg/sql/sem/builtins/builtins.go

```diff
@@ -2407,7 +2407,8 @@ may increase either contention or retry errors, or both.`,
 	),
 
 	// https://www.postgresql.org/docs/9.6/functions-datetime.html
-	"timezone": makeBuiltin(defProps(),
+	"timezone": makeBuiltin(
+		tree.FunctionProperties{IgnoreVolatilityCheck: true},
```

Comment:
do you mind adding a comment linking to the issue here?
also wondering if this ignore should be at the overload level, leave that up to you though.

Reply:
Ya I think it should be at the overload level because that's where volatility is defined. I'll make the change.